src/umockdev.vala

```diff
@@ -745,6 +751,57 @@ public class Testbed: GLib.Object {
         this.ev_sender.send(devpath, action);
     }
 
+    /**
+     * umockdev_testbed_attach_ioctl:
+     * @self: A #UMockdevTestbed.
+     * @dev: (not nullable): Device path (/dev/...) to attach to.
```

Comment:
TIL that `(not nullable)` is [really a thing](https://gi.readthedocs.io/en/latest/annotations/giannotations.html#type-signature), I've just never seen it. That's the default and should also be clear for valac due to the `string` type?

Reply:
Oh, you are right! I thought I had double checked in the GIR that it was needed. But, turns out this happened due to the `?` and I actually forgot to drop the ? in `detach`.